javascript/operators/object_initializer.json

```
@@ -76,7 +76,7 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": true
+                "version_added": "0.1.100"
```

I think this one might be later (doesn't seem likely that this would be in such an early release of Node.js, but in such a late release of Chrome). Maybe 4.0.0? https://node.green/#ES2015-syntax-object-literal-extensions-computed-properties

```suggestion
                "version_added": "4.0.0"
```